appcues/src/debug/java/com/appcues/CompositionTestHelper.kt

```diff
@@ -47,33 +50,36 @@ fun ComposeContent(json: String, imageLoader: ImageLoader) {
     }
 }
 
+// This helper supports testing any arbitrary combination of traits with optional given content.
+// It constructs a synthetic 1-step experience. The traits are applied at the experience level. The
+// optional content is injected into the single step, if present.
 @Composable
-fun ComposeContainer(
-    context: Context,
-    experienceJson: String,
-    traitJson: String,
-    groupIndex: Int,
-    stepIndex: Int,
-    imageLoader: ImageLoader)
+fun ComposeContainer(context: Context, stepContentJson: String?, traitJson: List<String>, imageLoader: ImageLoader)
```

Comment:
I'm probably not getting it right, but I think that what @mmaatttt did [here](https://github.com/appcues/appcues-mobile-experience-spec/pull/168/files) is a bit different, it seems like traits where should refer to a different file in a source folder, that would contain the entire json for that trait name.

Reply:
in this case, `traitJson` is a list of JSON objects, one per trait, to be injected. They get deserialized and mapped down on line 65 below. Does that clarify?

Reply:
oh yeah sorry, after this one we need to update the ScreenshotTestExt.kt to support calling traits from separate files then